gontrib/contributions_test.go

```diff
@@ -0,0 +1,40 @@
+package gontrib
+
+import (
+	"os"
+	"testing"
+
+	"github.com/schollz/gontributions/vcs/mediawiki"
+	"github.com/schollz/gontributions/vcs/obs"
+)
+
+func TestScanContributions(t *testing.T) {
+	PullSources = true
+	configuration := Configuration{
+		Emails: []string{"jubalh@openmailbox.org", "g.bluehut@gmail.com"},
+		Projects: []Project{
+			{Name: "nudoku", Description: "Ncurses sudoku game", Gitrepos: []string{"https://github.com/jubalh/nudoku"}},
+			{Name: "profanity", Description: "Ncurses based XMPP client", URL: "http://profanity.im/", Gitrepos: []string{"https://github.com/boothj5/profanity"}},
+			{Name: "Funtoo", Description: "Linux distribution", URL: "http://funtoo.org/", Gitrepos: []string{"https://github.com/funtoo/ego", "https://github.com/funtoo/metro"}, MediaWikis: []mediawiki.MediaWiki{{BaseUrl: "http://funtoo.org", User: "jubalh"}}},
+			{Name: "openSUSE", Description: "Linux distribution", URL: "http://opensuse.org/", Obs: []obs.OpenBuildService{{Apiurl: "https://api.opensuse.org", Repo: "utilities/vifm"}}},
+		},
+	}
+
+	contributions, err := ScanContributions(configuration)
+	os.RemoveAll("repos-git")
+	os.RemoveAll("repos-obs")
+	if err != nil {
+		t.Errorf("Got an error scanning contributions: " + err.Error())
+	}
+
+	for i, contribution := range contributions {
```

Comment:
Here you only test for the git repos of nudoku, profanity and ego, right? But above you also add the wiki to the gontributions and the vifm OBS. Could you add those for checking here too?